src/styles/card_style.css

```diff
@@ -0,0 +1,82 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 auto 40px;
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.card__rating {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__link {
+  display: block;
+  box-sizing: border-box;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 38px;
+  text-align: center;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
```

Comment:
Group styles, read more - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/